Create IPO Chart

cyber-mountain-man opened the pull request:
Add IPO chart for Garden Catalog project

cyber-mountain-man committed:
Create IPO Chart

Add IPO chart for Garden Catalog project

amberfeeley reviewed (commented):
# AF1
<img width="2880" height="1800" alt="Screenshot 2025-08-21 at 10 49 05 PM" src="https://github.com/user-attachments/assets/de1e9336-6eb5-4578-8b69-7ef3eee4e580" />
It looks like I'm getting a syntax error :(

cyber-mountain-man committed:
Update IPO Chart

### 🔑 Changes Made
- Simplified the subgraph labels to `I`, `P`, `O` so Mermaid parses them cleanly.  
- Removed extra arrows inside the subgraphs (Mermaid didn’t like the earlier style).  
- Kept the **Inputs → Processes → Outputs** pipeline flow.

cyber-mountain-man commented:
Just made an update tell me how it looks?

amberfeeley commented:
# AF2
Testing for discussions

amberfeeley commented on IPO Chart:
```diff
@@ -0,0 +1,31 @@
+flowchart LR
+    %% Define columns as groups
+    subgraph I[Inputs]
+        A1[User Location (ZIP/GPS)]
```
# AF3
Thread test

amberfeeley replied:
Okay threads work directly on code. Good note for us to have!

cyber-mountain-man replied:
So odd. Lol

cyber-mountain-man commented:
## 🌱 IPO Chart – Garden Catalog Web Application

| **Input** | **Processing** | **Output** |
|-----------|----------------|------------|
| - User Location (ZIP/GPS)  <br> - User Preferences (plants, animals, land use)  <br> - Garden Parameters (size, soil, irrigation)  <br> - **External Data Sources:**  <br> &nbsp;&nbsp;• USDA Plant Hardiness Zones  <br> &nbsp;&nbsp;• Weather API (NWS/OpenWeather)  <br> &nbsp;&nbsp;• Soil Database (SSURGO / Web Soil Survey) | - Geolocation & Zone Lookup  <br> - Weather Forecast & Frost Risk Analysis  <br> - Soil Data Retrieval & Analysis  <br> - Plant Suitability Engine  <br> - Garden Planner & Calendar Builder  <br> - Acreage Calculator (plants/animals)  <br> - Data Storage (users, gardens, historical records) | - Plant Recommendations (Now / Indoors / Wait)  <br> - Personalized Garden Calendar  <br> - Soil & Land Suitability Report  <br> - Acreage & Land Planning Estimates  <br> - User Dashboard (saved gardens, progress tracking) |

cyber-mountain-man committed:
Update IPO Chart to Markdown

- Converted IPO Chart to Markdown table format for easier versioning
- Displays cleanly in GitHub README/docs

cyber-mountain-man commented:
Just made change in branch to Markdown for the fix.
